normandy/recipes/tests/test_exports.py

```diff
@@ -190,7 +192,26 @@ def test_check_connection(self, rs_settings, requestsmock):
             # restore write permissions
             requestsmock.get(collection_url, json=allow_write_payload)
 
+        # Signer version should be > 5.1.0.
```

Comment:
Should this be the below instead?

```suggestion
        # Signer version should be >= 5.1.0.
```